app/src/main/java/com/github/houseorganizer/houseorganizer/panels/MainScreenActivity.java

```diff
@@ -83,11 +92,43 @@ protected void onCreate(Bundle savedInstanceState) {
         findViewById(R.id.calendar_view_change).setOnClickListener(v -> calendar.rotateCalendarView(v, this, calendarAdapter, calendarEvents));
         findViewById(R.id.add_event).setOnClickListener(this::addEvent);
         findViewById(R.id.refresh_calendar).setOnClickListener(this::refreshCalendar);
-        findViewById(R.id.new_task).setOnClickListener(v -> TaskView.addTask(db, taskList, taskListAdapter, listView));
+        //findViewById(R.id.new_task).setOnClickListener(v -> TaskView.addTask(db, taskList, taskListAdapter, listView));
```

Comment:
why is this commented out ?

Reply:
This line should be deleted Now there is the function bottomAddButton that does this